datalad/core/local/save.py

```diff
@@ -309,7 +309,7 @@ def save_ds(args, version_tag=None):
                         # calls
                         paths=None,
                         # prevent whining of GitRepo
-                        git=True if not hasattr(ds.repo, 'annexstatus')
+                        git=True if not hasattr(ds.repo, 'uuid')
```

Comment:
I'd prefer to use that occasion to unify this kind of test:

```suggestion
                        git=True if not isinstance(ds.repo, AnnexRepo)
```

Reply:
Not a blocker, though.

Reply:
Let's do this elsewhere.